public/locales/de/app.json

```diff
@@ -83,6 +83,8 @@
   "Sending…": "Senden …",
   "Settings": "Einstellungen",
   "Share screen": "Bildschirm teilen",
+  "Show call feed debug info": "Call-Feed-Inspector anzeigen",
+  "Show VoIP debug info": "VoIP-Inspektor anzeigen",
```

Comment:
These files should only be edited by Weblate otherwise it causes conflicts

Reply:
Oh ok.  I will remove this lines